src/main/kotlin/com/avail/interpreter/levelTwo/operation/L2_UNREACHABLE_CODE.kt

```diff
@@ -29,20 +29,16 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
```

Comment:
```
/*
 * L2_UNREACHABLE_CODE.kt
...
```

Reply:
yup, 85% of the way through I realized I forgot to update these. Any recommendations on a regex to fix this up in one fell swoop? My regex-foo is weak sauce...

Reply:
For posterity, Todd slacked the solution, to replace:
```
    \b(\w+)\.java
```
with
```
    $1.kt
```
I have a regex that's slightly stronger, which seems to have fewer false-positives.  Set up a global replace of:
```
^(/\*\n \* \w+)\.java\b
```
with
```
$1.kt
```
BUT – make sure you enter `*.kt` in the File mask combo box in the top right, and select the checkmark.
Then scroll through the list of hits to make sure they're all ok, and scroll through them again to make sure they're all for Kotlin files.  If everything looks perfect, replace all.

In my regex, `^` matches being at the start of a line, then matches /* and a linefeed, then a space and * and another space, then one or more word characters.  All that falls into group $1.  Then match .java and no more word characters.  The replacement replaces all of that, including the .java, with that first group plus .kt on the end.